coral/src/domain/topic/topic-api.ts

```diff
@@ -66,4 +72,16 @@ const getTopicTeam = async ({
   );
 };
 
-export { getTopics, getTopicNames, getTopicTeam };
+const getTopicAdvanvedConfigOptions = (): Promise<
```

Comment:
Here's a small typo - "getTopicAdvan~~v~~**c**edConfigOptions". It extends through other files/namings.